sdk/storage/tests.yml

```diff
@@ -47,6 +50,19 @@ stages:
           Path: sdk/storage/platform-matrix.json
           Selection: sparse
           GenerateVMJobs: true
+      PostSteps:
+        - task: Maven@3
+          displayName: 'Run Stress Scenarios'
+          inputs:
+              mavenPomFile: sdk/storage/azure-storage-perf/pom.xml
+              options: -Dstorage-stress-scenarios
```

Comment:
I'd either add `$(DefaultOptions)` or `--no-transfer-progress`  to `options` to reduce the verbosity of this step.